server/protocols/jmap-rfc-8621/doc/specs/spec/mail/rights.mdown

```diff
@@ -0,0 +1,136 @@
+# urn:apache:james:params:jmap:mail:shares extension
+
+<aside class="warning">
+  Implemented
+</aside>
+
+This extension is specific to the Apache James server as it relies on a common shared storage allowing arbitrary cross-account entity access.
+
+This extension to the JMAP specification enables to:
+
+ - Grant access on mailboxes, and the email they contain to other users.
+ - Enable display of shared mailboxes, and shared messages as part of the main user account, making effectively account sharing transparent from a client perspective.
+
+## Defined capability
+
+The new capability `urn:apache:james:params:jmap:mail:shares` is defined.
+
+This extension defines both behavioral changes and additional fields for the Mailbox object.
+
+If specified, behavioral changes and additional fields defined hereafter MUST apply.
+
+If unspecified no additional fiel to Mailbox object are returned, and the behaviour need to be exactly the one of `urn:ietf:params:jmap:mail`.
+
+## Addition to existing data types
+
+The following data type are defined:
+
+`Username` of type `String` correspond to a username.
+
+`Right` of type `String` defines an action that can be carried out on a mailbox or its content.
+
+The following `Right`s are defined:
+
+ - `"a"` Administer: the user can view and modify the rights of this mailbox
```

Comment:
why not using `MailboxRights` as defined in the JMAP specification?

Reply:
Good question. I believe because we wanted to stick to the https://tools.ietf.org/html/rfc4314 wording, but I'm in favour of the change you ask (but as a separate task ;-) )